src/__tests__/verify.test.ts

```diff
@@ -13,28 +16,51 @@ describe('sigstore TUF', () => {
       }).not.toThrow();
     });
   });
-});
-
-describe('python TUF sample', () => {
-  const root = Metadata.fromJSON(MetadataKind.Root, roots.pythonSample);
-  const timestamp = Metadata.fromJSON(
-    MetadataKind.Timestamp,
-    timestamps.pythonSample
-  );
 
-  describe('verify root', () => {
-    it('verifies the root successfully', () => {
+  describe('verify timestamp meta from root ', () => {
+    it('verifies the timestamp metadata successfully', () => {
       expect(() => {
-        root.verifyDelegate('root', root);
+        root.verifyDelegate('timestamp', timestamp);
       }).not.toThrow();
     });
   });
 
-  describe('verify timestamp meta from root ', () => {
-    it('verifies the timestamp metadata successfully', () => {
+  describe('verify the role target from targets ', () => {
+    it('verifies the role metadata successfully', () => {
+      const delegator = Metadata.fromJSON(MetadataKind.Targets, targetsJson);
+      const newDelegate = Metadata.fromJSON(MetadataKind.Targets, role1Json);
+
       expect(() => {
-        root.verifyDelegate('timestamp', timestamp);
+        delegator.verifyDelegate('role1', newDelegate);
       }).not.toThrow();
     });
+
+    it('cannot find role', () => {
+      const delegator = Metadata.fromJSON(MetadataKind.Targets, targetsJson);
+      const newDelegate = Metadata.fromJSON(MetadataKind.Targets, role1Json);
+
+      expect(() => {
+        delegator.verifyDelegate('wrongrole', newDelegate);
+      }).toThrow('no delegation found for wrongrole');
```

Comment:
Same comment as above. For all of these error cases, let's check the error class instead of the specific error message.